labels.md

```diff
@@ -0,0 +1,220 @@
+# Goals
+
+* synchronise labels between repos
+* make labels easier to set automatically
+* make labels clearer
+* make it easier to tell at a glance what is blocking PRs from landing
+* make colours more useful by making them more uniform
+
+# Naming scheme
+
+## Current issues
+
+* TiDB uses a `category/label` scheme, but with some exceptions
+  - closed, component, priority, status, type
+* TiKV uses a `C: label` scheme
+  - C (component), D (difficulty), P (priority), S (status), T (type)
+* PD uses a `category/label` scheme, but with some exceptions
+  - area, backport, kind, priority, 
+
+Bots work on all repos, but bot-readable labels are not distinguished.
+
+There is some duplication within a repo and some inconsistency about the category a label belongs to.
+
+## Proposal
+
+We use a `C: label` scheme for all repos (I don't feel strongly about the scheme, as long as there is just one. `C: label` is shorter, `category/label` is easier to understand). The categories are: component, priority, status, type, and difficulty.
```

Comment:
I prefer the scheme of 'category/label'

Reply:
The problem is that labels are too long and too distractive. You can have a notable impression comparing [TiDB's PR list](https://github.com/pingcap/tidb/pulls) and [TiKV's](https://github.com/tikv/tikv/pulls).

Reply:
Any thoughts?